### ISSUE TITLE
Vʀᴇғɪɴᴛ of STM32F303

### ISSUE THREAD

**Issue by wocard2** [jankae/LCR#4]
Start.cpp says

	constexpr float ReferenceTypical = 1.2f;

but the typical value for the Internal reference voltage of the STM32F303 is 1.23 V according to the datasheet. Harmless.

**Comment by jankae** [comment]
You are right. Maybe I mistakenly used the min value? If you want, you can create a PR for this (and the other small issues you found), I'd be glad to merge that
